src/main/java/cz/startnet/utils/pgdiff/schema/PgSchema.java

```diff
@@ -418,7 +419,7 @@ public void addType(final PgType type) {
     /**
      * Returns a list of types
      *
-     * @return types List<PgType>
+     * @return types List of PgType
```

Comment:
Isn't the List<type> syntax more appropriate here?  It's common syntax in Java I think?

Reply:
Yes, this is the common syntax in Java, but if I use <PgType> I get a error because this is a unknow tag to javadoc. To solve this its necessary to include the tag PgType in pom.xml like I done with the todo tag, and I did not  do this to avoid fill the pom.xml with more changes

Reply:
thank you :)